Added table to migration, updated screenshots in the Subscriber Source topic (#740)
mulesoft/docs-connectors: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc

<anypoint-mq:subscriber config-ref="Anypoint_MQ_Config"
  destination="queue">
   <anypoint-mq:subscriber-type >

>       <anypoint-mq:polling fetchSize="20">

Reviewer: invalid value, the range is from 1 to 10. The default is 10.

<anypoint-mq:subscriber destination="queue"
    config-ref="Anypoint_MQ_Config">
    <anypoint-mq:subscriber-type>

>         <anypoint-mq:polling fetchSize="20">

Reviewer: invalid value, the range is from 1 to 10. The default is 10.

</anypoint-mq:subscriber>
----

This table lists the 2.x `polling` parameters and their 3.x equivalents.

[%header,cols="20a,30a,20a,30a"]
|===
| Version 2.x Parameter |Description| Version 3.x Equivalent |Description
|`fetchSize`| Declared at the `config` level, controls the size of the local message buffer, which is calculated as three times the `fetchSize` value.

> |`fetchSize`| Declared at the `Subscriber` level, controls the size of the local message buffer, which is calculated as three times the `fetchSize` value.

Reviewer: Please change to:
"Sets the maximum amount of messages to be fetched on each polling execution, in range from 1 to 10 messages"

This is based on the Javadoc of the parameter: 
https://github.com/mulesoft/anypoint-mq-connector/blob/develop-4.x/anypoint-mq-extension/src/main/java/com/mulesoft/extension/mq/api/source/PollingTypeSubscriberFactory.java#L57

</anypoint-mq:subscriber>
----

This table lists the 2.x `polling` parameters and their 3.x equivalents.

[%header,cols="20a,30a,20a,30a"]
|===
| Version 2.x Parameter |Description| Version 3.x Equivalent |Description
|`fetchSize`| Declared at the `config` level, controls the size of the local message buffer, which is calculated as three times the `fetchSize` value.
|`fetchSize`| Declared at the `Subscriber` level, sets the maximum number (1-10) of messages to fetch on each polling execution.
The default is 10.

> 

Reviewer 2: Just checking: both parameters named the same, right?

Author: Yes.

> |`fetchTimeout`| Controls the time waiting for a response from the service.

Reviewer 2: ```suggestion
|`fetchTimeout`| Limits the wait time for a response from the service.
```

|Deprecated| This parameter is no longer available. The connector always uses the maximum long-polling value.

|`frequency`| Controls when to recheck for messages after the queue is empty.

> |Deprecated | This parameter is no longer available. The connector handles the frequency internally, fixed at 1 second.

Reviewer 2: ```suggestion
|Deprecated | This parameter is no longer available. The connector uses a fixed frequency of 1 second.
```
